src/components/side-panel/panel-header.js

```diff
@@ -204,26 +204,43 @@ const defaultActionItems = [
 ];
 
 function PanelHeaderFactory() {
-  const PanelHeader = ({
+  return class PanelHeader extends Component {
+    static propTypes = {
+      appName: PropTypes.string,
+      version: PropTypes.string,
+      uiState: PropTypes.object,
+      uiStateActions: PropTypes.object,
+      logoComponent: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
+      actionItems: PropTypes.arrayOf(PropTypes.any)
+    };
+
+    static defaultProps = {
+      logoComponent: KeplerGlLogo,
+      actionItems: defaultActionItems
+    };
+
+    render() {
+      const {
         appName,
         version,
-        actionItems = defaultActionItems,
+        actionItems,
         onSaveMap,
         onExportImage,
         onExportData,
         onExportConfig,
-        logoComponent = KeplerGlLogo,
         visibleDropdown,
         showExportDropdown,
         hideExportDropdown
-      }) => (
+      } = this.props;
+
+      return (
         <StyledPanelHeader className="side-panel__panel-header">
           <StyledPanelHeaderTop className="side-panel__panel-header__top">
-            <logoComponent version={version}/>
+            <this.props.logoComponent appName={appName} version={version}/>
```

Comment:
PanelHeader can't be rendered as a pure functional component, because we can only use `this.props.logoComponent` syntax to render the local component, otherwise, react will interpret `logoComponent` as a tag